python-bindings/b2_terraform/provider_tool.py

```diff
@@ -26,6 +27,38 @@ def change_keys(obj, converter):
     return {converter(k).replace('__', '_'): v for k, v in obj.items()}
 
 
+def convert_json_to_go(obj: dict, keys_to_keep: dict):
```

Comment:
Maybe we can completely replace `change_keys` function with this one?

Reply:
Not yet, as change_keys is also used in preprocess to convert to camel case.